### ISSUE TITLE
Missing image error when signing into Medley Online

### ISSUE THREAD

**Issue by pamoroso** [Interlisp/medley#1836]
**Describe the bug**

Signing into Medley Online and clicking `Run Medley` leads to a blank page with the following error:

```
Unable to find image 'ghcr.io/interlisp/online-medley:production' locally docker: Error response from daemon: manifest unknown. See 'docker run --help'. 
```

**To Reproduce**

1. sign into your Interlisp Online account
2. click `Run Medley`

**Expected behavior**

A working Medley session.

**Screenshots**

N/A

**Context (please complete the following information):**

 - OS: Linux Mint Cinnamon
 - OS Version: 22
 - Host arch: x86_64 (System76 Merkaat)
 - Maiko version: N/A
 - IL:MAKESYSDATE: N/A

**Additional context**

The issue occurs also when signing into a guest Medley Online session.

FYI @fghalasz.

**Comment by fghalasz** [comment]
Fixed, sort of.  Online is now running a slightly older version of Medley (Jun 5).  Lets see how tonight's weekly build works to get a newer version online.

Not sure what went wrong but it probably had something to do with my installing a new development image to try out the recent Notecards changes.  Will track this issue going forward.

**Comment by pamoroso** [comment]
I confirm it's back online.

**Comment by fghalasz** [comment]
Root cause for this issue is now fixed in online repo PR#58 - fix oio-medley-pulldev command.  The pulldev command no longer deletes the production image when the development image = production image prior to pulling down the new development image.
